app/models/submission.rb

```diff
@@ -14,6 +14,9 @@ class Submission < ActiveRecord::Base
   validates :english_title, presence: true
   validates :english_abstract, presence: true
   validates :english_keywords, presence: true
+
+  after_create :notify_editors
```

Comment:
Trzeba zrobić to inaczej - w Pani kontrolerze można jeszcze usunąć zgłoszenie. Zatem to musi być zrobione w kontrolerze - dopiero wtedy, gdy użytkownik nie dodaje już więcej recenzentów.